changelogs/fragments/48-add_folder_template.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - folder_template_from_vm - add module and tests to create a template from an existing VM in vcenter and store the template in a folder
```

Comment:
Those kind of things usually read users not developers. I think users are not interested about if you added tests. Developers are, but developers read git commit message, so I think it's not necessary to note about tests in changelog.

Reply:
ah ok, makes sense! ill keep that in mind for the future